ui/pages/bridge/bridge.util.ts

```diff
@@ -124,7 +125,12 @@ export async function fetchBridgeTokens(
 
   tokens.forEach((token: unknown) => {
     if (
-      validateResponse<SwapsTokenObject>(TOKEN_VALIDATORS, token, url) &&
+      validateResponse<SwapsTokenObject>(
+        TOKEN_VALIDATORS.concat(TOKEN_AGGREGATOR_VALIDATORS),
+        token,
+        url,
+        false, // Don't log errors for tokens
```

Comment:
Ah yea I think I was seeing a ton of errors for tokens in the logs, thanks for this.